lib/main.js

```diff
@@ -335,7 +335,12 @@ const Hydrogen = {
 
   createResultBubble(editor: atom$TextEditor, code: string, row: number) {
     const { grammar, filePath, kernel } = store;
-    if (!grammar || !filePath) return;
+
+    if (!filePath || !grammar) {
+      return atom.notifications.addInfo(
```

Comment:
Should we make it a warning or error instead?

Reply:
Changed this, please squash!